### PR TITLE
General UI enhancements Part 1

### PR DESCRIPTION
**Pull request by peterszerzo**
Per https://trello.com/c/Jg3HDyuc/106-operational-ui-styling

### CODE REVIEW

**Comment by TejasQ** [comment]
UGH Can you please include the compiled artifacts in _separate_ commits?

File: packages/components/scripts/dev-server/site.tsx [hunk 1]
```diff
@@ -16,26 +23,19 @@ class Site extends React.Component<{}, {}> {
...
-              this.setState(p => ({
-                selected: n
-              }))
-            }}
-          />
->>>>>>> Implement API
+          <Card>
+            <Record>
+              <RecordSummary>
+                <Heading1Type>Deutsche Bahn (German Railway Company)</Heading1Type>
```
Comment (micha-f) [inline packages/components/scripts/dev-server/site.tsx]: Did you code on this in a train? :)

Comment (peterszerzo) [reply packages/components/scripts/dev-server/site.tsx]: That's exactly right - it was perfect time for some trivia research :)

**Comment by peterszerzo** [comment]
@TejasQ compiled artifacts can't go in separate commits unfortunately, lerna relies on them being around.

The alternative would be to make micro-commits for every single tweak, but that would result in having to review around 30 PR's that step on each other's toes. I'd rather get it done in one go..

File: packages/showcase/utils/index.js [hunk 2]
```diff
@@ -9,8 +9,15 @@ const repoBasePath = "https://rawgit.com/Contiamo/operational-ui/master"
  * request at build time.
  */
 export const fetchFromRepo = (path, startLine, endLine) => {
-  return Promise.resolve("<!-- separator --><pre><code>var a = 2;</code></pre><!-- separator -->")
-  // return fetch(`${repoBasePath}${path}`)
-  //   .then(res => res.text())
-  //   .then(res => startLine ? res.split("\n").slice(startLine, endLine).join("\n") : res)
+  return fetch(`${repoBasePath}${path}`)
```
Comment (TejasQ) [inline packages/showcase/utils/index.js]: Can we make this return implicit?

Comment (peterszerzo) [reply packages/showcase/utils/index.js]: Yes, absolutely, I've made a note of it for a future commit!

**Pull request merged**